Modules/_blake2/blake2b_impl.c

```diff
@@ -391,47 +391,26 @@ py_blake2b_dealloc(PyObject *self)
         PyThread_free_lock(obj->lock);
         obj->lock = NULL;
     }
+
+    PyObject *type = Py_TYPE(self);
```

Comment:
```suggestion
    PyTypeObject *type = Py_TYPE(self);
```

Same comment in the other file.